workshop/exercise-8/index.html

```diff
@@ -1 +1,57 @@
-<!-- ...crickets... -->
\ No newline at end of file
+<!-- ...crickets... -->
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1>Registration form</h1>
+    <form action="">
+     <div>
+      <div>
+       Username<input type="text" required name="username" />
+      </div>
```

Comment:
I would suggest using label tags like this <label>username</label> this will help us later when using javascript and we might want to associate a particular label with a particular input field. Other than that really nice job with whole workshop :)